### PR TITLE
feat: add landing page

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
## Screenshots
Screenshot (228)
## Deploy Preview Link: https://deploy-preview-1--devdeadviz-my-notes.netlify.app/

### CODE REVIEW

**Review by vikrantkode — APPROVED** [review]
- UI Looks good
- Code is clean
- You can go forward with this .

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
-import React from "react";
+import Mockman from "mockman-js";
+import { Route, Routes } from "react-router-dom";
+import { LandingPage } from "./pages";
 
 const App = () => {
-  return <div>App</div>;
+  return (
+    <>
+      <Routes>
```
Comment (bharati-21) [inline src/App.js:8]: Suggestion - This is fine for now. But later on, when the routes increase, you could move this to a separate component (say NavRoutes or WebsiteRoutes) and then render it here.

**Review by bharati-21 — APPROVED** [review]
Nice folder structure. I only had one suggestion that you can look into later as the app grows.

All looks good.

File: src/App.js (hunk 1 above)
Comment (devdeadviz) [reply src/App.js:8]: Okay Bharati, I will take care of this.

**Pull request merged**